@kiva/kv-components/vue/KvSwitch.vue

```diff
@@ -12,7 +12,6 @@
 				class="tw-sr-only tw-peer"
 				type="checkbox"
 				role="switch"
-				:value="value"
```

Comment:
This could be replaced with `:checked="checked"` so that the switch will be updated when `checked` changes

Reply:
Nice point! This component has some unused variables and code, like this

`model: {
     prop: 'checked',
     event: 'change',
},`

There is a prop called _checked_ ready to use, but I think is better to upgrade this component, in other ticket, to remove unused code.